ansible_collections/arista/avd/examples/l2ls-fabric/README.md

```diff
@@ -0,0 +1,413 @@
+
+```bash
+# current working directory: ~/ansible-avd-examples
+ansible-playbook arista.avd.install_examples
+```
+
+Output will show something similar to the following. If not, please make sure that AVD and all requirements are installed properly. Also verify that the collections_paths in the ansible.cfg file is correct.
+
+```shell
+ ~/ansible-avd-examples# ansible-playbook arista.avd.install_examples
+
+PLAY [Install Examples]***************************************************************************************************************************************************************************************************************************************************************
+
+TASK [Copy all examples to ~/ansible-avd-examples]*****************************************************************************************************************************************************
+changed: [localhost]
+
+PLAY RECAP
+****************************************************************************************************************************************************************************************************************************************************************************
+localhost                  : ok=1    changed=1    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0
+```
+
+After the playbook has run successfully, the following directory structure will be created.
+
+```shell
+ansible-avd-examples/     (directory where playbook was run)
+  ├── l2ls-fabric/
+    ├── documentation/
+    ├── group_vars/
+    ├── images/
+    ├── intended/
+    ├── switch-basic-configurations/
+    ├── ansible.cfg
+    ├── inventory.yml
+    ├── playbook.yml
+    └── README.md (this document)
+```
+
+???+ info
+
+    If the content of any file in the example is ***modified*** and the playbook is run again, the file ***will not*** be overwritten. However, if any file in the example is ***deleted*** and the playbook is run again, the file will be re-created.
+
+## Design Overview
+
+### Physical L2LS Topology
+
+The drawing below shows the physical topology used in this example. The interface assignment shown here are referenced across the entire example, so keep that in mind if this example must be adapted to a different topology.
+
+![Figure: 1](images/pure_L2LS_topo.svg)
+
+???+ note
+
+    The FW/L3 Device and individual hosts (A-D) in this example are not managed by AVD, but the switch ports connecting to these devices are.
+
+## Basic EOS Switch Configuration
+
+Basic connectivity between the Ansible controller host and the switches must be established before Ansible can be used to deploy configurations. The following should be configured on all switches:
+
+- A hostname configured purely for ease of understanding.
+- An IP enabled interface - in this example the dedicated out-of-band management interface is used.
+- A username and password with the proper access privileges
```

Comment:
Here we are mixing a list with punctuation or no punctuation. I would use one or the other but not both in the same list breakdown.

Reply:
@JulioPDX Agreed.  Thanks for pointing that out.  I modified the list.